pycds/util.py

```diff
@@ -1,5 +1,30 @@
 import pycds
 
+
+def check_migration_version(
+    executor,
+    schema_name=pycds.get_schema_name(),
+    # `version` must be kept up to date with latest migration
+    # a test checks it, however, in case you don't
+    version='8fd8f556c548',
+):
+    """Check that the migration version of the database schema is compatible
+    with the current version of this package.
+
+    This implementation is quick and easy, relying on manual updating of the
+    correct version number.
+    """
+    current = executor.execute(f"""
+        SELECT version_num 
+        FROM {schema_name}.alembic_version
+    """).scalar()
```

Comment:
I'm not sure I understand what this `scalar()` call is doing.  What is the default output?

Reply:
`scalar()` unwraps the SQLAlchemy `ResultProxy` value returned by `.execute()` into a single value when that is what is expected. It saves some hassle doing that manually.